packages/react-charting/src/components/SankeyChart/SankeyChart.base.tsx

```diff
@@ -7,15 +7,29 @@ import { SankeyGraph, SankeyLayout, sankey as d3Sankey, sankeyJustify, sankeyRig
 import { select, selectAll } from 'd3-selection';
 import { area as d3Area, curveBumpX as d3CurveBasis } from 'd3-shape';
 import * as React from 'react';
-import { ChartHoverCard, IBasestate, IChartHoverCardProps, SLink, SNode } from '../../index';
+import { IBasestate, SLink, SNode } from '../../types/IDataPoint';
```

Comment:
This change should be reverted. It is breaking the build.

![image](https://github.com/microsoft/fluentui/assets/98592573/9a807b5c-d00a-4d3a-a67d-85143b20abed)